com.github.inercia.k3x.json

```diff
@@ -0,0 +1,120 @@
+{
+  "app-id": "com.github.inercia.k3x",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.36",
+  "sdk": "org.gnome.Sdk",
+  "command": "k3x",
+  "finish-args": [
+    "--device=dri",
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=wayland",
+    "--socket=ssh-auth",
+    "--share=network",
+    "--env=LD_LIBRARY_PATH=/app/lib:/app/lib/x86_64-linux-gnu:/lib64:/lib:/usr/lib64:/usr/lib:/usr/local/lib",
```

Comment:
I don't think this is needed.

Reply:
Actually per https://github.com/elementary/granite/issues/241 it's the only way for us to make it work